coalib/results/Diff.py

```diff
@@ -192,6 +192,14 @@ def modified(self):
         return result
 
     @property
+    def has_changes(self):
+        """
+        :return: True if the modified file has real difference when
+                 compared with the original file, False otherwise.
+        """
+        return self.modified != self._file
+
+    @property
```

Comment:
I left the decorator from the `unified_diff` function as it is and added the decorator to `has_changes`. But I think that `git` found the first difference in line `195` because the lines `194` and `202` are the same.
Is it possible to make `git` to show the difference starting from line `194`?